### PR TITLE
removed r2c and replaced with Semgrep, Semgrep Inc or team at Semgrep…

### PR DESCRIPTION
**Pull request by s-santillan**
… phrasing

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

Does not include:

* FAQs, to be merged on monday
* Release notes, because historically that was our name at the time of release.

### CODE REVIEW

**Review by adamkvitek — APPROVED** [review]
@s-santillan thank you!

File: docs/contributing/updating-a-grammar.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Summary (ocaml-tree-sitter)
 In ocaml-tree-sitter:
 1. Update submodule tree-sitter-X.
 2. From `lang/`, run `./test-lang X`.
-3. From `lang/`, ask an r2c developer to run `./release X`.
+3. From `lang/`, ask an Semgrep team developer to run `./release X`.
```
Comment (s-santillan) [inline docs/contributing/updating-a-grammar.md:17]:
```suggestion
3. From `lang/`, ask a Semgrep team developer to run `./release X`.
```

File: docs/faq.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ The [Semgrep Registry](https://semgrep.dev/explore) contains rules from differen
 
 [Semgrep Code](https://semgrep.dev/products/semgrep-code)’s Team tier includes Pro rules which are proprietary and only available to paying customers.
 
-### Is it ok to run Semgrep or the r2c rules on my work projects?
+### Is it ok to run Semgrep or the Semgrep, Inc rules on my work projects?
```
Comment (s-santillan) [inline docs/faq.md:33]:
```suggestion
### Is it ok to run Semgrep or Semgrep, Inc rules on my work projects?
```

File: docs/index.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ Semgrep is developed and commercially supported by [r2c, a software security com
 
 <SupportedLanguagesTable />
 
-To determine experimental, beta, or general availability (GA) status, r2c scans a wide corpus of projects and measure the parse rate of each language. For more details see [the breakdown of all supported languages](supported-languages/).
+To determine experimental, beta, or general availability (GA) status, Semgrep scans a wide corpus of projects and measure the parse rate of each language. For more details see [the breakdown of all supported languages](supported-languages/).
```
Comment (s-santillan) [inline docs/index.md:67]:
```suggestion
To determine experimental, beta, or general availability (GA) status, the Semgrep team scans a wide corpus of projects and measure the parse rate of each language. For more details see [the breakdown of all supported languages](supported-languages/).
```

File: docs/semgrep-code/getting-started.md [hunk 4]
```diff
@@ -360,6 +360,6 @@ You can onboard (add) your entire organization's users and repositories by creat
 
 ### Getting support
 
-Refer to [Troubleshooting Semgrep Cloud Platform](/docs/troubleshooting/semgrep-app/) for common installation issues. Help is also available for all users through the [r2c Community Slack](https://r2c.dev/slack).
+Refer to [Troubleshooting Semgrep Cloud Platform](/docs/troubleshooting/semgrep-app/) for common installation issues. Help is also available for all users through the [Semgrep Community Slack](https://semgrep.slack.com).
```
Comment (s-santillan) [inline docs/semgrep-code/getting-started.md:363]:
```suggestion
Refer to [Troubleshooting Semgrep Cloud Platform](/docs/troubleshooting/semgrep-app/) for common installation issues. Help is also available for all users through the [Semgrep Community Slack](https://go.semgrep.dev/slack).
```

**Pull request merged**